lib/datadog/core/diagnostics/environment_logger.rb

```diff
@@ -13,6 +13,10 @@ def log_configuration!(prefix, data)
           logger.info("DATADOG CONFIGURATION - #{prefix} - #{data}")
         end
 
+        def log_debug!(prefix, data)
+          logger.debug("DATADOG CONFIGURATION - #{prefix} - #{data}")
```

Comment:
s/CONFIGURATION/DEBUG/ ?

Reply:
The default logger format already includes the level:

> ``D, [2024-07-16T15:54:39.386684 #327838] DEBUG -- datadog: [datadog] (dd-trace-rb/lib/datadog/core/diagnostics/environment_logger.rb:17:in `log_debug!') DATADOG CONFIGURATION - TRACING INTEGRATIONS - {"action_cable_analytics_enabled":"false"``

so I'm not sure it's worth repeating in the string? Not a very strongly-held opinion though.

Reply:
I meant that inside the method, the assumption appears to be that this is debug logging for configuration, but is it possible for clients to log it something else at debug level which wouldn't be configuration?

Reply:
The environment logger was built to gather and log configuration; when would it get used for something other than configuration? :thinking: